src/ScrollBox/ScrollBox.js

```diff
@@ -3,13 +3,13 @@ import cx from 'classnames'
 import Box from '../Box/Box'
 import './scrollbox.css'
 
-const ScrollBox = ({className, children, ...props}) => {
+const ScrollBox = ({className, scrollX = false, scrollY = false, children, ...props}) => {
   const classes = cx('u-scrollbox', className)
 
   return (
-    <Box className={classes} {...props}>
-      <Box className='u-scrollbox__wrapper'>
-        <Box className='u-scrollbox__container'>{children}</Box>
+    <Box width="full" overflow="hidden" className={classes} {...props}>
```

Comment:
needs `position="relative" too

Reply:
i have the overflow classes adding position relative but ill remove it to make it explicitly necessary to add it here so it's less behind the scenes stuff